drivers/place/visitor_mailer.cr

```diff
@@ -312,6 +315,60 @@ class Place::VisitorMailer < PlaceOS::Driver
     )
   end
 
+  def template_fields : Array(TemplateFields)
+    time_now = Time.utc.in(@time_zone)
+    common_fields = [
+      {name: "visitor_email", description: "Email address of the visiting guest"},
+      {name: "visitor_name", description: "Full name of the visiting guest"},
+      {name: "host_name", description: "Name of the person hosting the visitor"},
+      {name: "host_email", description: "Email address of the host"},
+      {name: "building_name", description: "Name of the building where the visit occurs"},
+      {name: "event_title", description: "Title or purpose of the visit"},
+      {name: "event_start", description: "Start time (e.g., #{time_now.to_s(@time_format)})"},
+      {name: "event_date", description: "Date of the visit (e.g., #{time_now.to_s(@date_format)})"},
+      {name: "event_time", description: "Time of the visit (or 'all day' for 24-hour events)"},
+    ]
+
+    invitation_fields = common_fields + [
+      {name: "room_name", description: "Name of the room or area being visited"},
+      {name: "network_username", description: "Network access username (if network credentials enabled)"},
+      {name: "network_password", description: "Generated network access password (if network credentials enabled)"},
```

Comment:
We could make this more dynamic and only include the user/pass if `send_network_credentials` is enabled.